docs/src/less/base/index.less

```diff
@@ -200,6 +200,17 @@ main > div > h2 {
     margin: 8px;
 }
 
+.demo__signal {
```

Comment:
I don't think we need this demo css. Hotness signals should all line up inline by default (i.e. inline-block). At most we might need to apply a bit of horizontal margin for the demo if wanting to display them all on one line.